### ISSUE TITLE
enable GeoJSON output by default

### ISSUE THREAD

**Issue by steko** [totalopenstation/totalopenstation#54]
It should be possible to enable GeoJSON output by default without any additional external dependencies.

GeoJSON defaults to WGS84 coordinate reference system. A standard-compatible way to express local coordinate systems must be found, because it is rather common when working with total stations that are not equipped with GPS.

**Comment by steko** [comment]
Fixed in dde25e60017540d9cc27d66b237e5396016b77df

**Comment by steko** [comment]
I should have upgraded all input formats to use `FeatureCollection` instead of a list of `Feature` objects.

**Comment by steko** [comment]
GeoJSON export is broken on master:

```
totalopenstation-cli-parser.py -i sample_data/leica_gsi/leica_gsi16_gurob.gsi -f leica_gsi -t geojson
Traceback (most recent call last):
  File "/home/steko/code/totalopenstation/env/bin/totalopenstation-cli-parser.py", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/steko/code/totalopenstation/scripts/totalopenstation-cli-parser.py", line 186, in <module>
    main(infile)
  File "/home/steko/code/totalopenstation/scripts/totalopenstation-cli-parser.py", line 183, in main
    sys.stdout.write(output.process())
  File "/home/steko/code/totalopenstation/totalopenstation/output/tops_geojson.py", line 56, in process
    return json.dumps(self.data.__geo_interface__)
AttributeError: 'list' object has no attribute '__geo_interface__'
```

**Comment by steko** [comment]
Fixed in 32758d08bfe1b553a06dd7b68d25258a3df2feb3.
